packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -450,7 +454,8 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
 
         // Add data store's attributes to the summary.
         const { pkg } = await this.getInitialSnapshotDetails();
-        const attributes = createAttributes(pkg, this.isRootDataStore, this.disableIsolatedChannels);
+        const isRootOrAliasedDataStore = await this.isRoot();
```

Comment:
I'd use isRoot for name of variable here.
If you feel like some semantic is not clear due to name, then likely same applies to function name

Reply:
Not sure if you are planning to address this or not but there are some inconsistencies in naming:
This variable for instance should be called isRootDataStore or the function it gets the value from should be called isRootOrAlias.
Similarly, setRoot sets isAliasedDataStore. Either the function should be called setAliasDataStore or the member should be called isRootDataStore (or isRootOrAlias).